### PR TITLE
chore: update packages & use improved-yarn-audit

### PR DESCRIPTION
**Pull request by dleard**

### CODE REVIEW

File: ggircs-app/app/components/Layout/PageRedirectHandler.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const PageRedirectHandler: React.FunctionComponent<Props> = ({
         environment,
         sessionQuery,
         {}
-      );
+      )  as unknown as PageRedirectHandlerQueryResponse;
```
Comment (dleard) [inline ggircs-app/app/components/Layout/PageRedirectHandler.tsx:42]: not sure why this all of a sudden started being angry, but casting it to the expected type appears to have fixed the issue.
"Type error: Property 'session' does not exist on type 'RelayObservable<PageRedirectHandlerQueryResponse>'."

File: ggircs-app/app/server/schema.graphql [hunk 2]
```diff
@@ -948,6 +948,11 @@ type Query implements Node {
   session: JwtToken
 }
 
+"""
```
Comment (dleard) [inline ggircs-app/app/server/schema.graphql:951]: not sure why this moved?

**Review by pbastia — APPROVED** [review]
Looks good to me!

File: ggircs-app/app/server/schema.graphql (hunk 2 above)
Comment (pbastia) [reply ggircs-app/app/server/schema.graphql:951]: This is odd, and it generated a huge change in the schema.json file...I wonder what could have caused that

**Pull request merged**